README.md

```diff
@@ -355,6 +355,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [any_hash.cr](https://github.com/Sija/any_hash.cr) - Recursive Hash with better JSON::Any included
  * [atomic_write.cr](https://github.com/chris-huxtable/atomic_write.cr) - Library for writing or apppending files atomically
  * [burocracia.cr](https://github.com/vnbrs/burocracia.cr) - burocracia.cr the dependecyless shard to validate, generate and format Brazilian burocracias such as CPF, CNPJ and CEP
+ * [bitfields](https://github.com/elorest/bitfields) - Pure Crystal implementation of BitFields. Handles encoding/decoding of bytes.
```

Comment:
I can suggest moving it to [Algorithms and Data structures](https://github.com/veelenga/awesome-crystal#algorithms-and-data-structures) or leave it here.

Please fix the specs.